### PR TITLE
Cleanup imports

### LINKED ISSUE #35: Improve orchestrator dependencies
* Fix the ordering in dependencies in all Cargo files to follow the same pattern
* Validate whether we need all of them and remove redundant

### PR DESCRIPTION
**Pull request by AaronKutch**
This contains changes that would make removing `num256` much easier. Also fixes #35.

### CODE REVIEW

File: orchestrator/test_runner/Cargo.toml [hunk 1]
```diff
@@ -10,28 +10,25 @@ name = "test-runner"
...
-orchestrator = {path = "../orchestrator/"}
 prost = "0.9"
 prost-types = "0.9"
 rand = "0.8"
-serde = "1.0"
-serde_derive = "1.0"
+serde = { version = "1", features = ["derive"] }
 serde_json = "1.0"
-tokio = {version = "1.4", features = ["macros", "rt-multi-thread"]}
+tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
```
Comment (dzmitryhil) [inline orchestrator/test_runner/Cargo.toml:32]: why the "tokio" version is "1" instead of "1.4" ?

**Review by dzmitryhil — CHANGES_REQUESTED** [review]

**Comment by dzmitryhil** [comment]
Also check the "checks" some of them are failed

File: orchestrator/ethereum_gravity/src/logic_call.rs [hunk 2]
```diff
@@ -193,7 +201,7 @@ fn encode_logic_call_payload(
         sig_arrays.sigs,
         Token::Struct(struct_tokens.to_vec()),
     ];
-    let payload = clarity::abi::encode_call(
+    let payload = gravity_utils::clarity::abi::encode_call(
```
Comment (leviathanbeak) [inline orchestrator/ethereum_gravity/src/logic_call.rs:204]: why not include just clarity::abi instead ?

File: orchestrator/ethereum_gravity/src/submit_batch.rs [hunk 3]
```diff
@@ -165,7 +170,7 @@ fn encode_batch_payload(
         batch.token_contract.into(),
         batch.batch_timeout.into(),
     ];
-    let payload = clarity::abi::encode_call("submitBatch((address[],uint256[],uint256,uint256,address),(uint8,bytes32,bytes32)[],uint256[],address[],uint256[],uint256,address,uint256)",
+    let payload = gravity_utils::clarity::abi::encode_call("submitBatch((address[],uint256[],uint256,uint256,address),(uint8,bytes32,bytes32)[],uint256[],address[],uint256[],uint256,address,uint256)",
```
Comment (leviathanbeak) [inline orchestrator/ethereum_gravity/src/submit_batch.rs:173]: same here

File: orchestrator/gbt/src/keys/mod.rs [hunk 4]
```diff
@@ -1,12 +1,13 @@
 pub mod register_orchestrator_address;
```
Comment (leviathanbeak) [inline orchestrator/gbt/src/keys/mod.rs:2]: you can clean up these empty lines

**Review by leviathanbeak — CHANGES_REQUESTED** [review]
seems fine, I noticed there are a lot of empty lines between multiple ```use``` statements, that could be cleaned up and removed.

Also there is big difference between tokio 1.0 and 1.4 so that should not be touched

File: orchestrator/ethereum_gravity/src/logic_call.rs (hunk 2 above)
Comment (AaronKutch) [reply orchestrator/ethereum_gravity/src/logic_call.rs:204]: This is from reexporting. I have just moved the import to the start of the file

File: orchestrator/gbt/src/keys/mod.rs (hunk 4 above)
Comment (AaronKutch) [reply orchestrator/gbt/src/keys/mod.rs:2]: We could also use the `One` setting https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#group_imports, but I have always personally used `StdExternalCrate`.

File: orchestrator/test_runner/Cargo.toml (hunk 1 above)
Comment (AaronKutch) [reply orchestrator/test_runner/Cargo.toml:32]: "1" is different from "1.0", "1.0" would get minor version 0 but "1" gets the latest minor version. After this change the Cargo.lock version is `1.17.0`. Tokio `1.4` is really old, see how many fixes there have been: https://github.com/tokio-rs/tokio/blob/master/tokio/CHANGELOG.md.

Comment (AaronKutch) [reply orchestrator/test_runner/Cargo.toml]: I have done it

File: orchestrator/gbt/src/keys/mod.rs (hunk 4 above)
Comment (AaronKutch) [reply orchestrator/gbt/src/keys/mod.rs:2]: I presume you are ok

**Review by dzmitryhil — APPROVED** [review]

**Review by leviathanbeak — APPROVED** [review]

**Pull request merged**